index.html

```diff
@@ -199,9 +199,9 @@ <h3>Downloads</h3>
                         <p><strong>Customize your download:</strong></p>
                         <ul>
                             <li>Set title</li>
-                            <li>Choose between video, audio or captions download</li>
-                            <li>Select the most suitable file format</li>
-                            <li>Download the resolution you need</li>
+                            <li>Video or audio only, with or without captions</li>
+                            <li>Many file formats</li>
+                            <li>In any resolution</li>
```

Comment:
```suggestion
                            <li>In any available resolution</li>
```

Reply:
This doesn't mean anything now. Not that it was great before. Can you get a list of all available resolutions?

Reply:
Taken from here: https://support.google.com/youtube/answer/6375112
    2160p: 3840x2160
    1440p: 2560x1440
    1080p: 1920x1080
    720p: 1280x720
    480p: 854x480
    360p: 640x360
    240p: 426x240